url/Cargo.toml

```diff
@@ -2,7 +2,7 @@
 
 name = "url"
 # When updating version, also modify html_root_url in the lib.rs
-version = "2.5.2"
+version = "2.5.3"
```

Comment:
Make sure to also update the version [here](https://github.com/servo/rust-url/blob/08a326820a4a54d6a9730ecabd590942fc6b66dc/url/src/lib.rs#L146)

Reply:
Thanks. Done.